<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CircuitOptions</class>
 <widget class="QWidget" name="CircuitOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CircuitOptions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Circuit Values</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="resistorLabel">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QTextEdit" name="resistorText">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="resistorScale">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Ω</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>kΩ</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MΩ</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="inductorLabel">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>L</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="capacitorLabel">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>C</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTextEdit" name="inductorText">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QTextEdit" name="capacitorText">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="inductorScale">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>mH</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>H</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>kH</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="capacitorScale">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>nF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>μF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>F</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="circuitConfigLabel">
           <property name="minimumSize">
            <size>
             <width>98</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Circuit Configuration</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="circuitConfigSelection">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Series</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Parallel</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="measureAcrossLabel">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Measure Across</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="measureAcrossSelection">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>R</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>L</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="simulateButton">
         <property name="text">
          <string>Simulate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Voltage Input</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="frequencyLabel">
           <property name="text">
            <string>Frequency</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="phaseLabel">
           <property name="text">
            <string>Phase</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="phaseUnits">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>radians</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>degrees</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="frequencyScale">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Hz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>kHz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MHz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>rad/s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>k rad/s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>M rad/s</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="voltageLabel">
           <property name="text">
            <string>Voltage</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QTextEdit" name="voltageText">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTextEdit" name="frequencyText">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="voltageScale">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>mV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>V</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vpp</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QTextEdit" name="phaseText">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="offsetLabel">
           <property name="text">
            <string>Offset</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QTextEdit" name="offsetText">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="signalTypeLabel">
           <property name="minimumSize">
            <size>
             <width>55</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Signal Type</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="signalTypeSelection">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Sinusoidal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Square</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
